<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>992</width>
                <height>601</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="windowOpacity">
            <double>1.000000000000000</double>
        </property>
        <property name="autoFillBackground">
            <bool>false</bool>
        </property>
        <property name="styleSheet">
            <string notr="true"/>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,2">
                        <item>
                            <widget class="QPushButton" name="pushButton_2">
                                <property name="text">
                                    <string>PushButton</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pushButton">
                                <property name="text">
                                    <string>PushButton</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QTreeView" name="treeView">
                                <property name="styleSheet">
                                    <string notr="true">QWidget {
                                        background-color: #1b1b22;
                                        }
                                    </string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="code"/>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QTabWidget" name="tabWidget">
                        <widget class="QWidget" name="tab">
                            <attribute name="title">
                                <string>Tab 1</string>
                            </attribute>
                        </widget>
                        <widget class="QWidget" name="tab_2">
                            <attribute name="title">
                                <string>Tab 2</string>
                            </attribute>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
        <action name="actionCloseWindow">
            <property name="text">
                <string>CloseWindow</string>
            </property>
            <property name="toolTip">
                <string>Close</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
